packages/components/src/components/post-closebutton/post-closebutton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Element, h, Host, Prop } from '@stencil/core';
+import { version } from '@root/package.json';
+
+/**
+ * @slot default - The post-closebutton is a slot in itself to be used easily in other components.
+ */
+@Component({
+  tag: 'post-closebutton',
+  shadow: false,
+})
+export class PostClosebutton {
+  @Element() host: HTMLPostClosebuttonElement;
+
+  /**
+   * The a11y label to use for the close button.
+   */
+  @Prop() readonly label?: string = 'Close button';
+
+  render() {
+    return (
+      <Host data-version={version} slot="post-closebutton">
+        <button class="btn btn-icon-close">
+          <post-icon aria-hidden="true" name="2043"></post-icon>
```

Comment:
Design has changed since I've made my PR and is not currently using the tokens that were previously defined. I asked Alessio to change them so I'll check afterwards that it looks as intended.